module/Database/src/Model/SubDecision/OrganRegulation.php

```diff
@@ -155,71 +155,62 @@ public function setChanges(bool $changes): void
         $this->changes = $changes;
     }
 
+    protected function getTemplate(): string
+    {
+        return 'Het %TYPE%reglement van %NAME% door %AUTHOR%, versie %VERSION% van %DATE% wordt %APPROVAL%%CHANGES%.';
+    }
+
+    protected function getAlternativeTemplate(): string
+    {
+        return 'The %TYPE%regulation of %NAME% by %AUTHOR%, version %VERSION% dated %DATE% is %APPROVAL%%CHANGES%.';
```

Comment:
Currently, this results in the following:
```tex
\decision[The committee regulation of TEST by Timo de Teststudent, version 1.0 dated 1 January 2000 is approved with mentioned changes.]{Het commissiereglement van TEST door Timo de Teststudent, versie 1.0 van 1 januari 2000 wordt goedgekeurd met genoemde wijzigingen.}
```

The [internal regulations](https://gewis.nl/en/association/regulations/ir/) went for the plural variant.

```suggestion
        return 'The %TYPE%regulations of %NAME% by %AUTHOR%, version %VERSION% dated %DATE% are %APPROVAL%%CHANGES%.';
```

Reply:
Ah didn't know that, I will fix it in a bit.

Reply:
Fixed in 39e3680103f38d42e8d5b1af06dc14f8cce74e4a.